app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.tsx

```diff
@@ -9,6 +10,17 @@ type AppProps = {
 	currentStage: string;
 };
 
+const mapCreateStatus = (currentStage: unknown): Status => {
```

Comment:
```suggestion
const mapCreateStatus = (currentStage: string): Status => {
```

Reply:
@rachellerathbone Was there a reason why this was unknown?

Reply:
Not sure why this was `unknown`. @rachellerathbone wrote this originally so I assumed it was an AtlasKit quirk. Looking at it now and `string` works correctly. Updated the code.